src/threading/Manager.cc

```diff
@@ -36,6 +38,127 @@ Manager::~Manager() {
         Terminate();
 }
 
+void Manager::InitPostScript() {
+    static auto get_message_thread_stats = []() -> const BucketedMessages* {
+        if ( ! thread_mgr->terminating ) {
+            double now = util::current_time();
+            if ( thread_mgr->bucketed_messages_last_updated < now - 1 ) {
+                thread_mgr->current_bucketed_messages.pending_in_total = 0;
+                thread_mgr->current_bucketed_messages.pending_out_total = 0;
+                for ( auto& m : thread_mgr->current_bucketed_messages.pending_in )
+                    m.second = 0;
+                for ( auto& m : thread_mgr->current_bucketed_messages.pending_out )
+                    m.second = 0;
+
+                MsgThread::Stats thread_stats;
+                for ( const auto& t : thread_mgr->msg_threads ) {
+                    t->GetStats(&thread_stats);
+
+                    thread_mgr->current_bucketed_messages.sent_in_total += thread_stats.sent_in;
+                    thread_mgr->current_bucketed_messages.sent_out_total += thread_stats.sent_out;
+                    thread_mgr->current_bucketed_messages.pending_in_total += thread_stats.pending_in;
+                    thread_mgr->current_bucketed_messages.pending_out_total += thread_stats.pending_out;
+
+                    for ( auto idx : pending_bucket_brackets ) {
+                        if ( thread_stats.pending_in <= idx ) {
```

Comment:
If you compare `<=` here, then we should change the label to `le` rather than `lt`. Will make a comment there, too.

Reply:
Switched it to `<` and left the label as `lt`. It's basically just so all of the zero values fall into the `1` bucket.

Reply:
Oh, I wasn't explicit enough. The Prometheus histogram uses the "le" label for buckets, so I figured staying consistent with that would've been nice.